src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -64,6 +64,28 @@ const en = ({ isDivorce, userCase, partner }: CommonContent) => ({
     } after your application has been checked.
     This will have the time, date and court your conditional order will be pronounced.`,
   },
+  subHeading1:
+    userCase.coClarificationUploadDocuments || userCase.coClarificationResponses
+      ? 'Latest information'
+      : 'What you need to do',
+  clarificationSubmitted: {
+    withDocuments: {
+      line1: `You have provided the information requested by the court. You'll receive an email by ${dayjs(
+        userCase.dateSubmitted
```

Comment:
I don't think that'll be correct. Can you use the due date and not calculate the 16 days?

Reply:
Added due date to line 1.

Reply:
Please check that works - I don't know for sure the dueDate is set by that event. It should be though. Also I think the date will still need formatting